docs/running-on-openshift-cluster.md

```diff
@@ -1,12 +1,14 @@
-# Running on an OpenShift `ROSA` cluster
+# Running on an OpenShift cluster
 
 ## Overview
 
-This document covers running an instance of `AnsibleAIConnect` on OpenShift `ROSA`. 
+This guide shows you how to deploy AnsibleAIConnect or AnsibleMCPServer on
+an OpenShift cluster.
 
-It assumes you have an OpenShift `ROSA` cluster running.
+These instructions assume you're using 
+[a Red Hat OpenShift Service on AWS (ROSA) cluster](https://docs.openshift.com/rosa/welcome/index.html)), 
+but any steps not specific to ROSA should also work on other OpenShift clusters.
```

Comment:
I would suggest to be generic here, saying it:
"These instructions assume you're using Red Hat OpenShift cluster running, if not, you can use [a Red Hat OpenShift Service on AWS (ROSA) cluster](https://docs.openshift.com/rosa/welcome/index.html) or other OpenShift clusters that you prefer"

wdyt?

Reply:
@de1987 I have pushed a new commit. Please re-review the updated portions on next Monday :-) 
Have a nice weekend!